src/main/resources/assets/js/application/interviewer/tdprInterviewerModuleConfig.js

```diff
@@ -16,7 +18,21 @@ define(['angular' 
                         controller: "tdprInterviewerHomeController" 
                     } 
                 }
-            }); 
+            }).state("tdpr.interviewer.details", {
+              url: "/{id}/details",
+              views: {
+                  "@": {
+                      templateUrl: "/html/partials/interviewer/tdpr-interviewer-details.html",
+                      controller: "tdprInterviewerDetailsController"
+                  }
+              },
+              resolve:{
+                  service: "tdprPersonService",
+                  promise: function(service, $stateParams){
+                      return service.getPersonDetails($stateParams.id);
+                  }
```

Comment:
we are resolving a person - it should look like this:
resolve: {
  person: function(tdprPersonService, $stateParams) {
    return: tdprPersonService.getPersonDetails($stateParams.personId);
  }
}